### PR TITLE
Release 5.6.0

### PR DESCRIPTION
**Pull request by nharper285**
## Summary of the Pull Request

_What is this about?_
Pull Request tracking the release of OneFuzz 5.6.0. 

## Info on Pull Request

_What does this include?_
Version and Changelog updates. 

## Validation Steps Performed

_How does someone test & validate?_
- [ ] Integration Tests.

### CODE REVIEW

**Review by tevoinea — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,15 +4,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.6.0
+### Added
+* Service: Add Function App Settings to Bicep Template and deploy.py. [#1973](https://github.com/microsoft/onefuzz/pull/1973)
+* Agent: Add Timestamp to Agent Log. [#1972](https://github.com/microsoft/onefuzz/pull/1972)
+
+### Changed
+* Agent/Supervisor/Proxy: Rename Supervisor to Agent. [#1989](https://github.com/microsoft/onefuzz/pull/1989)
```
Comment (mgreisen) [inline CHANGELOG.md:13]:
```suggestion
* Agent/Supervisor/Proxy: Rename the supervisor process from `onefuzz-supervisor` to `onefuzz-agent`. [#1989](https://github.com/microsoft/onefuzz/pull/1989)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,15 +4,34 @@ All notable changes to this project will be documented in this file.
...
+* Agent/Supervisor/Proxy: Rename Agent to Task. [#1980](https://github.com/microsoft/onefuzz/pull/1980)
```
Comment (mgreisen) [inline CHANGELOG.md:14]:
```suggestion
* Agent/Supervisor/Proxy: Rename the agent process that runs on the VM from `onefuzz-agent` to `onefuzz_task`. [#1980](https://github.com/microsoft/onefuzz/pull/1980)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,15 +4,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.6.0
+### Added
+* Service: Add Function App Settings to Bicep Template and deploy.py. [#1973](https://github.com/microsoft/onefuzz/pull/1973)
+* Agent: Add Timestamp to Agent Log. [#1972](https://github.com/microsoft/onefuzz/pull/1972)
```
Comment (mgreisen) [inline CHANGELOG.md:10]:
```suggestion
* Agent: Add a timestamp to Agent log to make it easier to correlate events. [#1972](https://github.com/microsoft/onefuzz/pull/1972)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,15 +4,34 @@ All notable changes to this project will be documented in this file.
...
+* Agent/Supervisor/Proxy: Rename Agent to Task. [#1980](https://github.com/microsoft/onefuzz/pull/1980)
+* Agent/Supervisor/Proxy: Ensure `GlobalFlag` Reg Value is Initialized for Target. [#1960](https://github.com/microsoft/onefuzz/pull/1960)
+* Agent/Supervisor/Proxy: Set Backtrace to Full. [#1959](https://github.com/microsoft/onefuzz/pull/1959)
+* Agent/Supervisor/Proxy: Updated multiple third-party Rust dependencies. [#2003](https://github.com/microsoft/onefuzz/pull/2003), [#2002](https://github.com/microsoft/onefuzz/pull/2002), [#1999](https://github.com/microsoft/onefuzz/pull/1999), [#1992](https://github.com/microsoft/onefuzz/pull/1992), [#1986](https://github.com/microsoft/onefuzz/pull/1986), [#1983](https://github.com/microsoft/onefuzz/pull/1983), [#1982](https://github.com/microsoft/onefuzz/pull/1982), [#1981](https://github.com/microsoft/onefuzz/pull/1981), [#1985](https://github.com/microsoft/onefuzz/pull/1985), [#1974](https://github.com/microsoft/onefuzz/pull/1974), [#1969](https://github.com/microsoft/onefuzz/pull/1969), [#1965](https://github.com/microsoft/onefuzz/pull/1965),
```
Comment (mgreisen) [inline CHANGELOG.md:17]:
```suggestion
* Agent/Supervisor/Proxy: Updated multiple third-party Rust dependencies. [#2003](https://github.com/microsoft/onefuzz/pull/2003), [#2002](https://github.com/microsoft/onefuzz/pull/2002), [#1999](https://github.com/microsoft/onefuzz/pull/1999), [#1992](https://github.com/microsoft/onefuzz/pull/1992), [#1986](https://github.com/microsoft/onefuzz/pull/1986), [#1983](https://github.com/microsoft/onefuzz/pull/1983), [#1982](https://github.com/microsoft/onefuzz/pull/1982), [#1981](https://github.com/microsoft/onefuzz/pull/1981), [#1985](https://github.com/microsoft/onefuzz/pull/1985), [#1974](https://github.com/microsoft/onefuzz/pull/1974), [#1969](https://github.com/microsoft/onefuzz/pull/1969), [#1965](https://github.com/microsoft/onefuzz/pull/1965)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,15 +4,34 @@ All notable changes to this project will be documented in this file.
...
+* Agent/Supervisor/Proxy: Set Backtrace to Full. [#1959](https://github.com/microsoft/onefuzz/pull/1959)
+* Agent/Supervisor/Proxy: Updated multiple third-party Rust dependencies. [#2003](https://github.com/microsoft/onefuzz/pull/2003), [#2002](https://github.com/microsoft/onefuzz/pull/2002), [#1999](https://github.com/microsoft/onefuzz/pull/1999), [#1992](https://github.com/microsoft/onefuzz/pull/1992), [#1986](https://github.com/microsoft/onefuzz/pull/1986), [#1983](https://github.com/microsoft/onefuzz/pull/1983), [#1982](https://github.com/microsoft/onefuzz/pull/1982), [#1981](https://github.com/microsoft/onefuzz/pull/1981), [#1985](https://github.com/microsoft/onefuzz/pull/1985), [#1974](https://github.com/microsoft/onefuzz/pull/1974), [#1969](https://github.com/microsoft/onefuzz/pull/1969), [#1965](https://github.com/microsoft/onefuzz/pull/1965), 
+* CLI/Service: Updated multiple first-party and third-party Python dependencies. [#2009](https://github.com/microsoft/onefuzz/pull/2009), [#1996](https://github.com/microsoft/onefuzz/pull/1996)
+
+### Fixed 
+* Agent: Removing Stray Print from task_logger. [#1975](https://github.com/microsoft/onefuzz/pull/1975)
```
Comment (mgreisen) [inline CHANGELOG.md:21]:
```suggestion
* Agent: Remove stray print statement from the task_logger. [#1975](https://github.com/microsoft/onefuzz/pull/1975)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,15 +4,34 @@ All notable changes to this project will be documented in this file.
...
+* Agent: Fix Local Coverage Definition. [#1962](https://github.com/microsoft/onefuzz/pull/1962)
```
Comment (mgreisen) [inline CHANGELOG.md:22]:
```suggestion
* Agent: Fix local coverage definition by removing a duplicated command line parameter. [#1962](https://github.com/microsoft/onefuzz/pull/1962)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,15 +4,34 @@ All notable changes to this project will be documented in this file.
...
+* Build: Fix YML Dependabot Formatting. [#1998](https://github.com/microsoft/onefuzz/pull/1998)
```
Comment (mgreisen) [inline CHANGELOG.md:23]:
```suggestion
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -4,15 +4,34 @@ All notable changes to this project will be documented in this file.
...
+* Service: Bug Fix in TimerTasks Function. [#1961](https://github.com/microsoft/onefuzz/pull/1961)
```
Comment (mgreisen) [inline CHANGELOG.md:24]:
```suggestion
```

**Review by mgreisen — APPROVED** [review]

File: CHANGELOG.md [hunk 9]
```diff
@@ -4,15 +4,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.6.0
+### Added
+* Service: Add Function App Settings to Bicep Template and deploy.py. [#1973](https://github.com/microsoft/onefuzz/pull/1973)
```
Comment (ranweiler) [inline CHANGELOG.md:9]:
```suggestion
* Service: Add Function App settings to Bicep template and `deploy.py`. [#1973](https://github.com/microsoft/onefuzz/pull/1973)
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -4,15 +4,32 @@ All notable changes to this project will be documented in this file.
...
+* Agent: Add a timestamp to Agent log to make it easier to correlate events. [#1972](https://github.com/microsoft/onefuzz/pull/1972)
```
Comment (ranweiler) [inline CHANGELOG.md:10]:
```suggestion
* Agent: Add a timestamp to agent log to make it easier to correlate events. [#1972](https://github.com/microsoft/onefuzz/pull/1972)
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -4,15 +4,32 @@ All notable changes to this project will be documented in this file.
...
+
+### Changed
+* Agent/Supervisor/Proxy: Rename the supervisor process from `onefuzz-supervisor` to `onefuzz-agent`. [#1989](https://github.com/microsoft/onefuzz/pull/1989)
```
Comment (ranweiler) [inline CHANGELOG.md:13]: Note for future changelogs: we longer have a "supervisor". We should probably start referring to the new `onefuzz-agent` as "the agent" and `onefuzz-task` as the task worker / executor.

File: CHANGELOG.md [hunk 12]
```diff
@@ -4,15 +4,32 @@ All notable changes to this project will be documented in this file.
...
+* Agent/Supervisor/Proxy: Rename the agent process that runs on the VM from `onefuzz-agent` to `onefuzz_task`. [#1980](https://github.com/microsoft/onefuzz/pull/1980)
```
Comment (ranweiler) [inline CHANGELOG.md:14]:
```suggestion
* Agent/Supervisor/Proxy: Rename the task executor that runs on the VM from `onefuzz-agent` to `onefuzz-task`. [#1980](https://github.com/microsoft/onefuzz/pull/1980)
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -4,15 +4,32 @@ All notable changes to this project will be documented in this file.
...
+* Agent/Supervisor/Proxy: Ensure `GlobalFlag` Reg Value is Initialized for Target. [#1960](https://github.com/microsoft/onefuzz/pull/1960)
```
Comment (ranweiler) [inline CHANGELOG.md:15]:
```suggestion
* Agent/Supervisor/Proxy: Ensure `GlobalFlag` registry value is initialized for targets. [#1960](https://github.com/microsoft/onefuzz/pull/1960)
```

File: CHANGELOG.md [hunk 14]
```diff
@@ -4,15 +4,32 @@ All notable changes to this project will be documented in this file.
...
+* Agent/Supervisor/Proxy: Set Backtrace to Full. [#1959](https://github.com/microsoft/onefuzz/pull/1959)
```
Comment (ranweiler) [inline CHANGELOG.md:16]:
```suggestion
* Agent/Supervisor/Proxy: Enable full backtraces on Rust panics. [#1959](https://github.com/microsoft/onefuzz/pull/1959)
```

File: CHANGELOG.md (hunk 11 above)
Comment (nharper285) [reply CHANGELOG.md:13]: Shall I make the edit for this round? or wait until next release?

Comment (ranweiler) [reply CHANGELOG.md:13]: Next is fine! 👍

**Review by ranweiler — APPROVED** [review]

**Pull request closed without merging**